esque/consumergroup.py

```diff
@@ -97,7 +97,9 @@ def get_consumer_offsets(self, group_coordinator, consumer_id, topic_assignment,
                     max(old_max, topic_offsets[partition_id][1] - current_offset),
                 )
 
-            return new_consumer_offsets
+            description_template = "{0} (min, max over {1} partitions)"
+            topic_description = description_template.format(topic.decode('utf-8'), len(consumer_offsets[topic].items()))
```

Comment:
You could do that in one pass with a f-string

```suggestion
            topic_description = f"{topic.decode('utf-8')} (min, max over {len(consumer_offsets[topic].items()} partitions"```